doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    etoh = smiles2atoms("CCO")
</pre><p>I don't find etoh to be a clear variable name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p><code>EtOH</code> is the most commonly used abbreviation for ethanol, but I renamed in to <code>ethanol</code></p>